lib/minimap-linter-binding.js

```diff
@@ -57,12 +74,14 @@ export default class MinimapLinterBinding {
 
   decorateMarker(marker, message) {
     const severity = messageSeverity(message);
-    const minimapMarkerDecoration = this.minimap.decorateMarker(marker, {
-      type: this.markerType,
-      class: `.linter-${severity}`,
-      plugin: 'linter',
-    });
-    this.decorations.add(minimapMarkerDecoration);
+    if (this.markerType[severity] !== 'off') {
+      const minimapMarkerDecoration = this.minimap.decorateMarker(marker, {
+        type: this.markerType[severity],
```

Comment:
Not all of the `observe` callbacks could have fired when there are messages available, so all the severities may not be set yet. Uou need to provide a fallback type here (`line`).

Reply:
@Arcanemagus I believe this means an issue in the old code? If the `observe`s were not fired, then `type: this.markerType` would evaluate to `undefined` in the old code?
Anyway, I will fix this in the constructor.

Reply:
Hah, yeah, it looks like the existing code had the same issue 😆